linker/Linker.Steps/MarkStep.cs

```diff
@@ -1715,6 +1724,47 @@ void MarkSomethingUsedViaReflection (string reflectionMethod, Action<Collection<
 			}
 		}
 
+		void MarkTypeUsedViaReflection (Collection<Instruction> instructions)
+		{
+			for (var i = 0; i < instructions.Count; i++) {
+				var instruction = instructions [i];
+
+				if (!CheckReflectionMethod (instruction, "GetType"))
+					continue;
+
+				_context.Tracer.Push ($"Reflection-{instruction.Operand as MethodReference}");
+				var typeAssemblyQualifiedName = OperandOfNearestInstructionBefore<string> (i, OpCodes.Ldstr, instructions);
+
+				//We're not going to support preserving generic types yet
+				if (typeAssemblyQualifiedName == null || typeAssemblyQualifiedName.Contains ("`"))
```

Comment:
Use `IndexOf ('`') < 1`

Reply:
changed to typeAssemblyQualifiedName.IndexOf('`') >= 0